Firebase/Core/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v6.3.1 -- M57
+- [fixed] Fixed race condition in component container. (#3967)
```

Comment:
Do we think this also fixes #3924?

Reply:
Tentatively, yes - good call. Will add it